### DISCUSSION TITLE
Condition not met when extra space exists in the IF part

### DISCUSSION THREAD

**Discussion by lxnunes** [Budibase/budibase#3753] [Help] [answered]
Morning!
I have noticed that when adding a _hide component_ condition, if an extra space exists in the IF clause (before of after), the condition is not enforced. In my case I was trying to hide a container if the user belongs to BASIC but with the extra space shown below, it did not work.

I have tested in my own environment, is this an expected behaviour?

![2021-12-10_09-22-13](https://user-images.githubusercontent.com/57442647/145552188-bb335fbb-7531-437b-9912-8c2c739c08a4.png)

cheers

**Comment by mjashanks** [comment, marked as answer]
Morning! Thanks for reporting. This is definitely not expected. I will log this as an issue

**Reply by lxnunes** [reply]
thanks @mjashanks
